validate/validate.go

```diff
@@ -424,7 +424,11 @@ func (v *Validator) CheckPlatform() (msgs []string) {
 		}
 	}
 
-	if v.platform != "windows" {
+	if v.platform == "windows" {
+		if v.spec.Windows == nil {
+			msgs = append(msgs, "'windows' MUST be set when platform is `windows`")
```

Comment:
This is [not true in rc5][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc5/config.md#L325-L326